ansible/roles/baselayout/vars/main.yml

```diff
@@ -71,6 +71,21 @@ packages: {
     'ccache,git,gmake,sudo,python3,py36-pip'
   ],
 
+  'macos10.10': [
+    'python@2',
+    'python'
+  ],
+
+  'macos10.11': [
+    'python@2',
+    'python'
+  ],
+
+  'macos10.12': [
```

Comment:
Is it worthwhile to add __macos10.13__ and __macos10.14__?  Does Ansible allow for a more generic __macos10.x__ ?

Reply:
no, we add new systems here as they are added to our infra, we don't have 10.13 yet although @refack was playing with it

Reply:
In the wild most Mac users turn on autoupdate or do manual updates so testing on macOS 10.14 would be more important than on other OSes.

Reply:
yes, but we have infra limitations that are stopping our march beyond 10.13, for now anyway